internal/build/testdata/TestBuildkitPrinter/echo-hi-success-verbose.master.txt

```diff
@@ -1,6 +1,8 @@
 [1/2] FROM docker.io/library/busybox@sha256:c94cf1b87ccb80f2e6414ef913c748b105060debda482058d2b8d0fce39f11b9
-[1/2] done | 702ms
+[1/2] FROM docker.io/library/busybox@sha256:c94cf1b87ccb80f2e6414ef913c748b105060debda482058d2b8d0fce39f11b9 [done: 2ms]
+[1/2] FROM docker.io/library/busybox@sha256:c94cf1b87ccb80f2e6414ef913c748b105060debda482058d2b8d0fce39f11b9 [done: 702ms]
```

Comment:
note: when these lines get printed in the web UI (and eventually in a terminal context), Tilt will overwrite the previous line